### ISSUE TITLE
compute_Mlincomb(nep::SPMF_NEP should be compute_Mlincomb!

### ISSUE THREAD

**Issue by jarlebring** [nep-pack/NonlinearEigenproblems.jl#89]
The code for `compute_Mlincomb(nep:SPMF_NEP`  in NEPCore:
```
    function compute_Mlincomb(
                        nep::SPMF_NEP,
                        λ::Number,
                        V::AbstractVecOrMat,
                        a::Vector=ones(size(V,2)))
```
at some point runs
```
    	V[:,findall(x->x==0,a)] .= 0
    	a[findall(x->x==0,a)] .= 1
```
which modifies `V` both and  `a`. Therefore, this should probably be a `compute_Mlincomb!`-function instead.

**Comment by jarlebring** [comment]
If we make fundamental changes in the core code like done here, I think we should aim to do pull requests so others can look at it. It was pure luck that I found it.

**Comment by meleg** [comment]
Fixed in my last commits 3e1461c69364417de4dc06beb54cf35d7328cddb and 8d085cad6fd6b738dcb25696f0da4b593080e97b
From now on big changes will go through pull requests.

**Comment by jarlebring** [comment]
:+1:
